inc/common/purge.php

```diff
@@ -420,16 +420,18 @@ function rocket_post_purge_urls_for_qtranslate( $urls ) {
  */
 function do_admin_post_rocket_purge_cache() { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
 	if ( isset( $_GET['type'], $_GET['_wpnonce'] ) ) {
-		$type = sanitize_key( $_GET['type'] );
+		$type       = sanitize_key( $_GET['type'] );
+		$type_array = explode( '-', $type );
 
-		$_type     = explode( '-', $type );
-		$_type     = reset( $_type );
-		$_id       = explode( '-', $type );
-		$_id       = end( $_id );
-		$_taxonomy = isset( $_GET['taxonomy'] ) ? sanitize_title( wp_unslash( $_GET['taxonomy'] ) ) : false;
+		$_type     = reset( $type_array );
+		$_id       = end( $type_array );
+		$_id       = is_numeric( $_id ) ? absint( $_id ) : 0;
+		$_taxonomy = isset( $_GET['taxonomy'] ) ? sanitize_title( wp_unslash( $_GET['taxonomy'] ) ) : '';
+		$_url      = '';
```

Comment:
What do you think about updating all those variables to not use the underscore prefix?

Reply:
Done in 0fe6507.